Shippo/APIResource.cs

```diff
@@ -381,9 +386,9 @@ public CarrierAccount RetrieveCarrierAccount (String object_id)
             return DoRequest<CarrierAccount> (ep, "GET");
         }
 
-        public ShippoCollection<CarrierAccount> AllCarrierAccount (Hashtable parameters)
+        public ShippoCollection<CarrierAccount> AllCarrierAccount ()
```

Comment:
Thanks for fixing this but unfortunately we can't break API contract once published even if it is incorrect unless we do a major version change. In this case let this function be there and have another that takes `parameters` and just calls this function by ignoring it.
```c#
public ShippoCollection<CarrierAccount> AllCarrierAccount (Hashtable parameters) {
     return AllCarrierAccount();
}
```